lms/lmsdb/models.py

```diff
@@ -375,6 +375,27 @@ def on_notification_saved(
         instance.delete_instance()
 
 
+class Tag(BaseModel):
```

Comment:
I now think about changing this tablename to ExerciseTag, and the connection to ExerciseToTag, in order to allow easy future expansion of the tags to user. IDK what the right approach to a state where we have both Users and Exercises that use tags, but I have a strong hunch that they shouldn't share the same table.